### ISSUE TITLE
Options on first start

### ISSUE THREAD

**Issue by licaon-kter** [syncthing/syncthing-android#622]
Can a menu be added for:
* (new user) Generate keys
* (reinstall etc) Import settings (and actually keys)

So one can bypass yet another key generation ?

**Comment by Nutomic** [comment]
We already had a discussion on the forum about a "welcome dialog", that explains how Syncthing works and so on. We could add an option to import keys there.

**Comment by gerroon** [comment]
Can you also show log window instead of "loading" which blocks the app use for long periods of sometimes

**Comment by Nutomic** [comment]
Startup time should be much better soon (in Syncthing v0.13 I think).

**Comment by Almtesh** [comment]
Actually, not, but I don't think it comes from syncthing.
I currently have a permanent loading screen. The only access I have to syncthing is the web interface through SSH.

**Comment by gerroon** [comment]
I agree with @Almtesh. I am not getting any kind of startup improvement with the latest stuff myself

**Comment by Catfriend1** [comment]
So let's agree import a backup should be made an option on the startup wizard. It should explain, that importing a backup is only allowed on one and the same device.

**Comment by imsodin** [comment]
Also add an option/hint to run syncthing at startup/in the background (and run conditions) as that's non-default and still a common setup.
